### PR TITLE
HBASE-22824 Show filesystem path for the orphans regions on filesystem

### PR DESCRIPTION
**Pull request by infraio**

### CODE REVIEW

**Comment by infraio** [comment]
About disabledTableRegions, it just is a refactor. It was introduced in previous issue. Move the implementation to generate inconsistent regions report.

File: hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp [hunk 1]
```diff
@@ -110,7 +110,7 @@
 
   <table class="table table-striped">
     <tr>
-      <th>Region</th>
+      <th>Region Encoded Name</th>
```
Comment (Apache9) [inline hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp]: I think through the path on fs we can know the table the region belongs to, but what about the start key? We can not get the start key through the encoded name?

Comment (infraio) [reply hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp]: Need change more code if we want start key here. And maybe a issue in future. Now all report is based on region encoded name. And the HBCK2 tool use encoded name as parameter too.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 40 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 355 | master passed |
| +1 | compile | 61 | master passed |
| +1 | checkstyle | 83 | master passed |
| +1 | shadedjars | 281 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 36 | master passed |
| 0 | spotbugs | 269 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 267 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 341 | the patch passed |
| +1 | compile | 58 | the patch passed |
| +1 | javac | 58 | the patch passed |
| +1 | checkstyle | 75 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 288 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1022 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 260 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 8467 | hbase-server in the patch passed. |
| +1 | asflicense | 24 | The patch does not generate ASF License warnings. |
| | | 12109 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/469 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 262f3176de14 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-469/out/precommit/personality/provided.sh |
| git revision | master / eb92b25652 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/1/testReport/ |
| Max. process+thread count | 4541 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 39 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 350 | master passed |
| +1 | compile | 58 | master passed |
| +1 | checkstyle | 81 | master passed |
| +1 | shadedjars | 291 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 265 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 263 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 324 | the patch passed |
| +1 | compile | 63 | the patch passed |
| +1 | javac | 63 | the patch passed |
| +1 | checkstyle | 91 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 288 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1017 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 38 | the patch passed |
| +1 | findbugs | 264 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 8934 | hbase-server in the patch passed. |
| +1 | asflicense | 24 | The patch does not generate ASF License warnings. |
| | | 12589 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/469 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 8c16a0d7d202 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-469@2/out/precommit/personality/provided.sh |
| git revision | master / c05ee7c9fe |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/2/testReport/ |
| Max. process+thread count | 4582 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-469/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**